### ISSUE TITLE
gradient Descent CCPP

### ISSUE THREAD

**Issue by rah07** [sabeelahmad/Gradient-Descent#1]
sir, please tell why we add-ones column in the dataset .please tell me a reason 

this is your code
------# ## Add row of ones to testing data
train_ones = np.ones(X_train.shape[0]).reshape(-1, 1)
X_train = np.append(X_train, train_ones, axis=1)
## Add row of ones to training data
test_ones = np.ones(X_test.shape[0]).reshape(-1, 1)
X_test = np.append(X_test, test_ones, axis=1)

and another question is that how we recognize which feature affects the cost and how?
